content/software.md

```diff
@@ -1,8 +1,19 @@
-<h2 id="software" class="section">Software for working with CWL <a id="Software_for_working_with_CWL" href="#Software_for_working_with_CWL" class="anchorfix longfix"><span>§</span></a></h2>
+---
+layout: page
+permalink: /software/
+title: Software
+redirect_from: /ecosystem/
+---
```

Comment:
This page could benefit from links to each of the sections at the top. Maybe hyperlinks within one sentence of prose?